### PR TITLE
Add Cilium documentation to networking.md

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

File: docs/networking.md [hunk 1]
```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
 
 In case of any issues the directory `/var/log/aws-routed-eni` contains the log files of the CNI plugin. This directory is located in all the nodes in the cluster.
 
+### Cilium Example for CNI and Network Policy
```
Comment (justinsb) [inline docs/networking.md]: We really need to split out this file by provider :-)

File: docs/networking.md [hunk 2]
```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
...
+```console
+$ export ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
+  --networking cilium\
+  --yes \
+  --name myclustername.mydns.io
```
Comment (justinsb) [inline docs/networking.md]: For the future, we try to stick with `example.com` - it's actually reserved for the purpose: https://www.iana.org/domains/reserved

File: docs/networking.md [hunk 3]
```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
...
+
+The following command sets up a cluster, in HA mode, with Cilium as the CNI and networking policy provider
+
+```console
+$ export ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
```
Comment (justinsb) [inline docs/networking.md]: Does it work if we do `kops create cluster --zones us-east-1c --networking cilium cilium.example.com` ?   As a reader, when I look at this, I'm wondering if cilium only works in HA, or only works on m4.larges etc...

File: docs/networking.md [hunk 4]
```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
...
+
+The following command will create your cluster with desired Cilium configuration
+
+```console
+$ kops update cluster myclustername.mydns.io --yes
+```
+
+#### Getting help with Cilium
+
+For problems with deploying Canal please post an issue to Github:
```
Comment (justinsb) [inline docs/networking.md]: I think you meant "For problems with deploying Cilium..." ;-)

**Comment by justinsb** [comment]
Thanks @nebril

I would definitely tweak it to do a default configuration, rather than HA/m4.large, because otherwise people might think Cilium needs those configurations.

/approve
/lgtm

File: docs/networking.md (hunk 4 above)
Comment (nebril) [reply docs/networking.md]: Whoops!

**Comment by nebril** [comment]
@justinsb thanks for review, your comments made a lot of sense, fixed.

**Comment by nebril** [comment]
@justinsb pinging for re-review :)

File: docs/networking.md [hunk 5]
```diff
@@ -42,6 +42,7 @@ Several different CNI providers are currently built into kops:
 * [romana](https://github.com/romana/romana)
 * [weave](https://github.com/weaveworks/weave-kube)
 * [amazon-vpc-routed-eni](./networking.md#amazon-vpc-backend)
+* [Cilium](http://docs.cilium.io/en/v1.1/)
```
Comment (techcet) [inline docs/networking.md]: @nebril you might want to just point to docs.cilium.io and let the URL redirect to the latest stable release.

File: docs/networking.md [hunk 6]
```diff
@@ -350,6 +351,73 @@ $ kops create cluster \
...
+#### Getting help with Cilium
+
+For problems with deploying Cilium please post an issue to Github:
+
+- [Cilium Issues](https://github.com/cilium/cilium/issues)
+
+For support with Cilium Network Policies you can reach out on Slack or Github:
+
+- [Cilium Github](https://github.com/cilium/cilium)
+- [Cilium Slack](https://cilium.slack.com)
```
Comment (techcet) [inline docs/networking.md]: @nebril this assumes the user already signed up for Slack. If they haven't, the screen says contact the Administrator. Let's save those people a step by putting this link to an invite: http://cilium.io/slack

**Comment by nebril** [comment]
@techcet thanks for review, addressed your comments.

**Comment by justinsb** [comment]
/approve
/lgtm

Thanks @nebril !

**Pull request merged**